src/ai/backend/gateway/manager.py

```diff
@@ -0,0 +1,128 @@
+import asyncio
+import enum
+import functools
+import json
+import logging
+import sqlalchemy as sa
+
+from aiohttp import web
+
+from ai.backend.common.logging import BraceStyleAdapter
+
+from .exceptions import InvalidAPIParameters, ServerFrozen
+from ..manager.models import kernels, KernelStatus
+
+
+log = BraceStyleAdapter(logging.getLogger('ai.backend.gateway.manager'))
+
+
+class ManagerStatus(enum.Enum):
+    RUNNING = 'running'
+    FROZEN = 'frozen'
+
+    @classmethod
+    def is_member(cls, value):
+        return any(value == item.value for item in cls)
```

Comment:
I think this is an unncessary boilerplate.
We can check this membership by checking if `ManagerStatus(some_string)` raises `ValueError` or not.